install/docker-compose/docker-compose.yml

```diff
@@ -9,20 +9,19 @@ services:
 
   keycloak:
     #image: jboss/keycloak:14.0.0
-    image: quay.io/microcks/microcks-keycloak:20.0.2
+    image: quay.io/keycloak/keycloak:21.0.2
```

Comment:
I didn't see it changed at first... It would be better to stick to 20.0.2 so that we'll use the same version everywhere.
Upgrading to `21.0.2` would be another topic and require more work.

Reply:
I have now changed the version back to 20.0.2. Made changes to the other docker compose files as well